### ISSUE TITLE
[FEATURE] Rename to AdminLTE 5 to keep in sync with Bootstrap

### ISSUE THREAD

**Issue by tacman** [ColorlibHQ/AdminLTE#3907]
Brainstorming: What do you think about renaming this release to AdminLTE 5, since it works with Bootstrap 5?  Yes, it would mean that there would be a missing (or alias) branch for version 4, but PHP survived without a version 6, so it's not unprecedented.

**Comment by Lovinity** [comment]
I've never been a fan of skipping versions or otherwise treating versions as arbitrary numbers in any way (such as symbolism).

Version numbers serve a very important purpose in the development community. They tell anyone who forks or otherwise uses the software/code when things change which could break their code extensions / modifications. Semantic Versioning was developed to standardize how we use version numbers in our software. And in my opinion, everyone should be using it because it makes logical sense and is easy for developers to understand.

Now, one argument could be that AdminLTE is skipping 4 and moving to 5. That's still a major version change just like if they were to have gone to version 4. So there's no harm done, right? Well, sort-of. I am implying based on this issue that, if AdminLTE uses version 5 to be consistent with Bootstrap's version, the developers will continue to match AdminLTE's version with Bootstrap's version as much as possible. This is a bad idea. If a breaking change occurs in AdminLTE, it should increment the major version, no exception, even if it is no longer matching Bootstrap's version. That way, those who use AdminLTE know the changes made may not be compatible with code written for previous versions of AdminLTE.

---

TL;DR, I do not think AdminLTE should skip versions even if others have done it. This breaks semantic versioning which is a very important standard in development.

**Comment by ZhangChengLin** [comment]
Don’t think the version information should be added to the official name

**Comment by makikata** [comment]
First a big **Thank you for your work all**  I dunno if my voice count, if not please ignore ;)

I agree with @Lovinity. I would like to add that linking its own versioning with one's dependency is not really the way to go IMHO. That might make sense if it were a library extending  bootstrap but this project uses bootstrap. So the other way around? Certainly not. 

Also what is the real goal of naming the release with bootstrap version?

- make it more obvious it is using bootstrap 5? Well users read the Changelog, Readme, docs ... if not they should :)
- trying to keep up with the pace of bootstrap releases? What if somebody makes an awesome contribution but breaks the API. Should you use a minor version, as mentioned above it is not wise. Or will you stall the project until bootstrap bumped to another major version? I do not think creating this kind of dependency will benefit the community.
